### PR TITLE
Allow Logging feature of ibm_is_lb to be controlled via terraform #2228

### PR DESCRIPTION
**Pull request by SunithaGudisagarIBM**
https://github.com/IBM-Cloud/terraform-provider-ibm/issues/2228

Screenshot 2021-02-25 at 9 43 52 PM

### CODE REVIEW

File: ibm/data_source_ibm_is_lb.go [hunk 1]
```diff
@@ -102,6 +102,13 @@ func dataSourceIBMISLB() *schema.Resource {
 				Description: "Load Balancer Host Name",
 			},
 
+			isLBLogging: {
+				Type:        schema.TypeBool,
+				Optional:    true,
```
Comment (hkantare) [inline ibm/data_source_ibm_is_lb.go:107]: In datasource its should be only computed (remove default and optional)

File: ibm/data_source_ibm_is_lb.go [hunk 2]
```diff
@@ -379,6 +386,7 @@ func lbGetByName(d *schema.ResourceData, meta interface{}, name string) error {
 		if *lb.Name == name {
 			d.SetId(*lb.ID)
 			d.Set(isLBName, *lb.Name)
+			d.Set(isLBLogging, *lb.Logging.Datapath.Active)
```
Comment (hkantare) [inline ibm/data_source_ibm_is_lb.go:389]: Add a condition to check if lb.Logging.!= nil && lb.Logging.Datapath != nil  {d.Set(isLBLogging, *lb.Logging.Datapath.Active)} and then set the value

File: website/docs/r/is_lb.html.markdown [hunk 3]
```diff
@@ -51,6 +51,7 @@ The following arguments are supported:
 * `type` - (Optional, Forces new resource, string) The type of the load balancer. Default value `public`. Supported values `public` and  `private`.
 * `resource_group` - (Optional, Forces new resource, string) The resource group where the load balancer to be created.
 * `tags` - (Optional, array of strings) Tags associated with the load balancer.
+* `logging` - (Optional, bool) Enable load balancer data logs for LogDNA. You can use data logs to help troubleshoot connectivity issues.
```
Comment (hkantare) [inline website/docs/r/is_lb.html.markdown:54]: simialrly add the entry for logging in datasource file also website/docs/d/is_lb.html.markdown

**Comment by SunithaGudisagarIBM** [comment]
@hkantare  Incorporated the review comments, Please Let me know if changes required.

**Pull request merged**